src/app/pages/common/entity/entity-table/entity-table.component.ts

```diff
@@ -1188,4 +1188,14 @@ export class EntityTableComponent<Row = any> implements OnInit, AfterViewInit, O
       this.expandedElement = this.expandedElement === element ? null : element;
     }
   }
+
+  isBasicColumnTemplate(column: string): boolean {
+    return !['expandedDetail', 'action', 'multiselect', 'expansion-chevrons'].includes(column) && !this.isInteractive(column);
+  }
+
+  isCustomColumnComponent(column: string): boolean {
+    const item = this.currentColumns.find((obj) => obj.prop === column);
+
+    return Boolean(!this.isBasicColumnTemplate(column) && item?.widget?.component && column == 'method');
+  }
```

Comment:
I wonder if just not using entity-table at all and using plain table would have been easier. Up to you though.

Reply:
That's a good idea. A lot of headaches are caused using entity-table on this page.